packages/fixed-decimal/src/FixedDecimal.ts

```diff
@@ -111,7 +113,7 @@ type FixedDecimalFormatOptions = {
  * @see {@link FixedDecimalFormatOptions}
  * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#moreprecision MDN Documentation for Intl.NumberFormat}
  */
-const defaultFormatOptions = (value: FixedDecimal) =>
+const defaultFormatOptions = (value: FixedDecimal<any>) =>
```

Comment:
I have this idea in my head that it's better to use `unknown` instead of `any` if possible?